src/rootStore.ts

```diff
@@ -1,4 +1,5 @@
 import { ObservableMap } from 'mobx';
 import { getGlobalContext } from './globalContext';
 
+export { ObservableMap };
```

Comment:
The no-unused-imports tslint rule was triggered on the ObservableMap, so I added an export.  I'm disabling that rule on that line instead.

---
In reply to: [126830331](https://github.com/Microsoft/satcheljs/pull/62#discussion_r126830331) [](ancestors = 126830331)